twake/backend/node/src/services/global-resolver.ts

```diff
@@ -0,0 +1,153 @@
+import { WorkspaceServiceImpl } from "./workspaces/services/workspace";
+import { UserExternalLinksServiceImpl } from "./user/services/external_links";
+import { UserNotificationBadgeService } from "./notifications/services/bages";
+import {
+  UserNotificationBadgeServiceAPI,
+  UserNotificationPreferencesAPI,
+} from "./notifications/api";
+import { DatabaseServiceAPI } from "../core/platform/services/database/api";
+import { NotificationPreferencesService } from "./notifications/services/preferences";
+import {
+  MessageThreadMessagesServiceAPI,
+  MessageThreadsServiceAPI,
+  MessageUserBookmarksServiceAPI,
+  MessageViewsServiceAPI,
+} from "./messages/api";
+import { ThreadMessagesService } from "./messages/services/messages";
+import { ThreadsService } from "./messages/services/threads";
+import { UserBookmarksService } from "./messages/services/user-bookmarks";
+import { SearchServiceAPI } from "../core/platform/services/search/api";
+import StorageAPI from "../core/platform/services/storage/provider";
+import { PubsubServiceAPI } from "../core/platform/services/pubsub/api";
+import { CounterAPI } from "../core/platform/services/counter/types";
+import { UserServiceImpl } from "./user/services/users/service";
+import { CompanyApplicationServiceImpl } from "./applications/services/company-applications";
+import { ApplicationServiceImpl } from "./applications/services/applications";
+import { ViewsServiceImpl } from "./messages/services/views";
+import { MessagesEngine } from "./messages/services/engine";
+import { FileServiceAPI } from "./files/api";
+import { FileServiceImpl } from "./files/services";
+import { ChannelServiceImpl } from "./channels/services/channel/service";
+import {
+  ChannelPendingEmailService,
+  ChannelService,
+  MemberService,
+  TabService,
+} from "./channels/provider";
+import { MemberServiceImpl } from "./channels/services/member/service";
+import ChannelPendingEmailServiceImpl from "./channels/services/channel/pending-emails/service";
+import { TabServiceImpl } from "./channels/services/tab";
+import AuthServiceAPI from "../core/platform/services/auth/provider";
+import { ConsoleServiceImpl } from "./console/service";
+import { StatisticsServiceImpl } from "./statistics/service";
+import { logger, TwakeServiceProvider } from "../core/platform/framework";
+import assert from "assert";
+
+type PlatformServices = {
+  realtime: RealtimeServiceAPI;
+  auth: AuthServiceAPI;
+  search: SearchServiceAPI;
+  storage: StorageAPI;
+  pubsub: PubsubServiceAPI;
+  counter: CounterAPI;
+};
+
+type CoreServices = {
+  workspaces: WorkspaceService;
+  companies: CompaniesServiceAPI;
+  users: UsersService;
+  console: ConsoleServiceAPI;
+  statistics: StatisticsAPI;
+  externalUser: UserExternalLinksService;
+  notificationBadge: UserNotificationBadgeServiceAPI;
+  notificationPreferences: UserNotificationPreferencesAPI;
+  messages: MessageThreadMessagesServiceAPI;
+  threads: MessageThreadsServiceAPI;
```

Comment:
Why not just import MessageServiceAPI ? It contains:
```
  userBookmarks: MessageUserBookmarksServiceAPI;
  threads: MessageThreadsServiceAPI;
  messages: MessageThreadMessagesServiceAPI;
  views: MessageViewsServiceAPI;
  engine: MessagesEngine;
```

Reply:
There is no more MessageServiceAPI  😬 
One of goals was to remove this grouping services, because of the over-use of the same services in different groups.

I told you today that I'm going to group some of them on level of global resolver  for convinience